columnphysics/icepack_therm_vertical.F90

```diff
@@ -629,21 +698,31 @@ subroutine frzmlt_bottom_lateral (dt,       ncat,     &
             enddo                  ! nilyr
 
             ! lateral heat flux, fside < 0
-            fside = fside + rside*etot/dt
+            fside = fside + rsiden(n)*etot/dt
 
          enddo                     ! n
 
       !-----------------------------------------------------------------
       ! Limit bottom and lateral heat fluxes if necessary.
-      ! FYI: fside is not yet correct for fsd, may need to adjust fbot further
+      ! Limit rside so we don't melt laterally more ice than frzmlt permits
+      ! FYI: fside is not yet correct for fsd, furthermore rside is
```

Comment:
Good catch.  The comments starting with "FYI" are helpful for the review but should be deleted before the PR is merged.